.claude/agents/code-reviewer.md

```
@@ -0,0 +1,50 @@
+---
+name: code-reviewer
+description: Reviews Python code for correctness, style, and climate_indices architectural compliance. Use when completing a story, when /review is invoked, or before merging.
+---
+
+You are a strict code reviewer for the `climate_indices` library.
```

markdownlint-cli2 (0.22.0)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)